deploy/examples/create-external-cluster-resources.py

```diff
@@ -1750,6 +1752,34 @@ def gen_shell_out(self):
         shOutIO.close()
         return shOut
 
+    def gen_nonsecured_data(self):
+        # TBD: add non-secured data keys
+        non_secured_data_keys = [
+            "ROOK_EXTERNAL_CEPH_MON_DATA",
+            "MONITORING_ENDPOINT",
+            "MONITORING_ENDPOINT_PORT",
+            "RADOS_NAMESPACE",
+            "RBD_POOL_NAME",
+            "RBD_METADATA_EC_POOL_NAME",
+            "TOPOLOGY_POOLS",
+            "TOPOLOGY_FAILURE_DOMAIN_LABEL",
+            "TOPOLOGY_FAILURE_DOMAIN_VALUES",
+            "RGW_ENDPOINT"
+        ]
+
+        non_secured_data = {}
+        if self._arg_parser.format == "bash":
+            non_secured_data = f"[NonSecuredData]\n"
+
+        for key in non_secured_data_keys:
+            if self.out_map[key]:
+                if self._arg_parser.format == "bash":
+                    non_secured_data += f"{key} = {self.out_map[key]}\n"
```

Comment:
can we follow same pattern as we did for user configmap values

Reply:
okay, updated. 
now the output will be 👇 
```
  {
    "name": "rook-ceph-external-intial-data",
    "kind": "ConfigMap",
    "data": "\"[InitialData]\nROOK_EXTERNAL_CEPH_MON_DATA = a=10.102.226.64:6789\nMONITORING_ENDPOINT = 10.244.0.19\nMONITORING_ENDPOINT_PORT = 9283\nRADOS_NAMESPACE = \nRBD_POOL_NAME = replicapool\nRBD_METADATA_EC_POOL_NAME = None\nTOPOLOGY_POOLS = \nTOPOLOGY_FAILURE_DOMAIN_LABEL = \nTOPOLOGY_FAILURE_DOMAIN_VALUES = \nRGW_ENDPOINT = \n\""
  }
  ```